acc_test/data/valid_opensearch_cluster_create.tf

```diff
@@ -22,11 +22,12 @@ resource "instaclustr_cluster" "validOpenSearch" {
     bundle  = "OPENSEARCH"
     version = "1.2.4"
     options = {
-      dedicated_master_nodes = false,
-      master_node_size       = "t3.small-v2",
-      data_node_size         = "t3.small-v2",
-      security_plugin        = true,
-      client_encryption      = true
+      dedicated_master_nodes  = false,
+      master_node_size        = "t3.small-v2",
+      data_node_size          = "t3.small-v2",
+      security_plugin         = true,
```

Comment:
Looking good to me. I think security plugin is always enabled for opensearch. As its not related to this ticket, probably we can update that later.

Reply:
thanks Cai,
security_plugin for opensearch only takes 'true' as value like you said, however, it's still considered as an input parameter.